pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+requires = ["setuptools>=63.0"]
```

Comment:
I believe you could declare dependency on `wheel` here as well and strike the separate instruction to install that package before building/installing Sopel

Reply:
Good point. I do not know why `wheel` hasn't already been added as an explicit dependency. Seems like I've already run into needing to install that manually when releasing some package version or another, maybe while Travis was broken.

Reply:
This was resolved, but the separate instruction still exists. (See other line note in CONTRIBUTING.)